modules/openapi-generator/src/main/resources/python/api_doc_example.mustache

```diff
@@ -34,5 +34,8 @@ from pprint import pprint
 {{#vendorExtensions.x-py-postponed-example-imports}}
 {{{.}}}
 {{/vendorExtensions.x-py-postponed-example-imports}}
-{{classname}}.update_forward_refs()
+try:
+    {{classname}}.{{#usePydanticV2}}model_rebuild{{/usePydanticV2}}{{^usePydanticV2}}update_forward_refs{{/usePydanticV2}}()
```

Comment:
You should use .model_rebuild(raise_errors=False) instead of try
However, this is a stopgap measure and in practice should be implemented without using raise_errors

Reply:
I think we can remove the `usePydanticV2` switch the template and the java class as `python` client generator now only supports pydantic v2.